patches/nginx-1.13.6-socket_cloexec.patch

```diff
@@ -0,0 +1,125 @@
+diff --git a/auto/unix b/auto/unix
+index 10835f6c..78a8073a 100644
+--- a/auto/unix
++++ b/auto/unix
+@@ -990,3 +990,14 @@ ngx_feature_test='struct addrinfo *res;
+                   if (getaddrinfo("localhost", NULL, NULL, &res) != 0) return 1;
+                   freeaddrinfo(res)'
+ . auto/feature
++
++ngx_feature="SOCK_CLOEXEC support"
++ngx_feature_name="NGX_HAVE_SOCKET_CLOEXEC"
++ngx_feature_run=no
++ngx_feature_incs="#include <sys/types.h>
++                  #include <sys/socket.h>"
++ngx_feature_path=
++ngx_feature_libs=
++ngx_feature_test="int fd;
++                  fd = socket(AF_INET, SOCK_STREAM, SOCK_CLOEXEC);"
++. auto/feature
```

Comment:
It's better to also test the availability of `fcntl(s, F_SETFD, FD_CLOEXEC)` here?